### PR TITLE
[docs] updates for YBA arm/graviton support

### PR DESCRIPTION
**Pull request by polarweasel**
Fixes for PLAT-5017

@netlify /preview/yugabyte-platform/configure-yugabyte-platform/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Ubuntu 18
+* Ubuntu 20
+
+### Python 3
+
+Python 3 is required. If you're using YugabyteDB Anywhere to provision nodes in public clouds, be sure the custom AMI you plan to use has Python 3 installed.
+
+### Oracle Linux notes
+
+Oracle Linux 8 assumes that either `gtar` or `gunzip` is present on the host AMI, and uses the `firewall-cmd` client to set default target ACCEPT.
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md:34]: To be completely correct, gtar/gunzip are requirements for _any_ os. It's just they got exposed in Oracle 8 testing. Perhaps we should have a specific heading for "custom AMI common requirements" or something like that. In there, we can list Python 3 (aliased as python3)/gtar/gunzip all as requirements on the custom AMI.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+YugabyteDB Anywhere support for Oracle Linux 8 has the following limitations:
+
+* Only the Red Hat Linux-compatible kernel is supported, to allow port changing. The Unbreakable Enterprise Kernel (UEK) isn't supported.
+* Systemd services aren't supported.
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md:39]: Systemd services are supported for Oracle/AlmaLinux 8 in 2.15.1+

**Review by chidmuthu — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* CentOS 7
+* Oracle Linux 8
+* Ubuntu 18
+* Ubuntu 20
+
+### Requirements for all OSes
+
+Python 3 is required. If you're using YugabyteDB Anywhere to provision nodes in public clouds, be sure the custom AMI you plan to use has Python 3 installed.
+
+Either `gtar` or `gunzip` must be present on the host AMI.
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md:32]: Did some more digging into the Ansible [module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/unarchive_module.html#notes) and we need all of the following: 
zipinfo
gtar

Unzip is not required as we don't use/distribute any zip files.

**Review by chidmuthu — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md (hunk 3 above)
Comment (polarweasel) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch.md:32]: Updated!

**Review by streddy-yb — APPROVED** [review]

**Pull request merged**